plugins/package-managers/python/src/main/kotlin/Poetry.kt

```diff
@@ -115,4 +118,14 @@ class Poetry(
     }
 }
 
-private val SCOPES_NAMES = listOf("main", "dev")
+private fun parseScopeNamesFromPyProject(pyprojectFile: File): List<String> {
```

Comment:
Consider passing `projectDir` as parameter, for more compact caller code.
It may also slightly help in the caller that the variable named `definitionFile` points to the lockfile, while the resolved siblings file points to the definition file.

Reply:
I'd prefer the function implementation not to make any assumptions about the file name, so I like to pass that explicitly.